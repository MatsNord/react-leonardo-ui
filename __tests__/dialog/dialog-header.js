import React from 'react';
import renderer from 'react-test-renderer';

import Dialog from '../../src/dialog';

test('Dialog header renders correctly', () => {
  const tree = renderer.create(
    <Dialog.Header>Header content</Dialog.Header>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Dialog header with title renders correctly', () => {
  const tree = renderer.create(
    <Dialog.Header>
      <Dialog.Title>
        Header title
      </Dialog.Title>
    </Dialog.Header>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Dialog header passes props correctly', () => {
  const tree = renderer.create(
    <Dialog.Header className="my-class" data-x="x" title="Header title">
      Header title
    </Dialog.Header>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
