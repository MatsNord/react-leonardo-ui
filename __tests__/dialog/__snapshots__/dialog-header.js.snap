// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog header passes props correctly 1`] = `
<div
  className="lui-dialog__header my-class"
  data-x="x"
  title="Header title"
>
  Header title
</div>
`;

exports[`Dialog header renders correctly 1`] = `
<div
  className="lui-dialog__header"
>
  Header content
</div>
`;

exports[`Dialog header with title renders correctly 1`] = `
<div
  className="lui-dialog__header"
>
  <div
    className="lui-dialog__title"
  >
    Header title
  </div>
</div>
`;
